psutil/TODO.aix

```diff
@@ -0,0 +1,30 @@
+AIX support is experimental and incomplete at this time.
+The following functions and methods are UNSUPPORTED on the AIX platform:
+
+    psutil.Process.memory_maps
+    psutil.Process.num_ctx_switches
+
+
+Process.io_counters read count is always 0
+
+
+TestSystemAPIs.test_pid_exists_2           there are pids in /proc that don't really exist
+TestProcess.test_name                      isolated python calls execve which changes process name
+TestProcess.test_num_fds                   opening a socket doesn't create fd in /proc/pid/fd (until data is sent??)
+TestProcess.test_open_files                /dev/null shows in open_files but it isn't a file
+TestProcess.test_pid_0                     pid 0 doesn't have a name on AIX
+TestProcess.test_prog_w_funky_name         funky name test doesn't work on UNIX environments
+TestProcess.test_zombie_process            trying to create a zombie process doesn't create a zombie process on AIX
```

Comment:
Do zombie processes exist on AIX? Is `status()` able to detect them?

Reply:
AFAIK they do exist and can be detected, but the trick we use to create one with `fork` doesn't work

Reply:
What happens when you call this test function?
https://github.com/giampaolo/psutil/blob/8b8da39e0c62432504fb5f67c418715aad35b291/psutil/tests/__init__.py#L335
Does it crash? Does it return a process but it's status is not zombie (what is it?)?

Reply:
This creates a process, but its status is 0 (SNONE) and not SZOMB. Then we crash here:

    call_until(lambda: zproc.status(), "ret == psutil.STATUS_ZOMBIE")

I'm not sure what that means.